NOTICE

```diff
@@ -92,6 +92,7 @@ Copyright (c) 2021 Lavinia Stiliadou
 Copyright (c) 2022 Anatoli Babenia
 Copyright (c) 2022 Miles Stötzner
 Copyright (c) 2022 Pawel Wójcik
+Copyright (c) 2022 Faisal Khan
```

Comment:
Please keep the alphabetical order and add yourself after Anatoli.

Reply:
fixed